apps/web/components/NewsItems/NewsItems.tsx

```diff
@@ -67,7 +67,7 @@ export const NewsItems = ({
           <Box display="flex" flexDirection="row" justifyContent="spaceBetween">
             <Text
               color={colorVariant === 'blue' ? 'blue600' : undefined}
-              variant="h3"
+              variant="h2"
```

Comment:
This will also affect every single organization page that's displaying their latest news at the bottom, I suggest we revert this and keep the variant as h3 for the lifeventsection title, news and categories on the frontpage or adding a way to change this with an explicit prop defaulting it to h3

![image](https://github.com/island-is/island.is/assets/43557895/f306ec81-5b25-41b7-af5b-e7d79607a82f)